apps/web-giddh/src/app/settings/profile/setting.profile.component.ts

```diff
@@ -852,30 +858,44 @@ export class SettingProfileComponent implements OnInit, OnDestroy {
         if (tabName === 'address') {
             this.loadAddresses('GET');
             this.loadLinkedEntities();
-            if (this.currentCompanyDetails && this.currentCompanyDetails.countryV2) {
-                this.loadTaxDetails(this.currentCompanyDetails.countryV2.alpha2CountryCode);
-                this.loadStates(this.currentCompanyDetails.countryV2.alpha2CountryCode);
-            }
+            this.loadTaxAndStates();
         }
         this.getPageHeading();
```

Comment:
function is useless since it works for mobile only. remove it

Reply:
Removed function call and function